css/base/base.css

```diff
@@ -0,0 +1,10 @@
+body {
+    text-align: center;
+}
+.container {
```

Comment:
Your indentation is inconsistent in your CSS. Not a big thing, but once you move into SASS CSS indentation is as important as JS and it's good to get into the habit now.